### PR TITLE
vault integration: add support for comments

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: frontend/app/components/crud-list/component.js [hunk 1]
```diff
@@ -3,18 +3,30 @@ import Ember from 'ember'
...
+        value: model[key],
+        edit: false
+      })
     })
-  })
+    .reduce((prev, curr) => {
+      if (curr.key.endsWith('_comment')) {
+        prev[prev.length - 1].comment = curr.value
+        return prev
+      } else {
```
Comment (kaldras) [inline frontend/app/components/crud-list/component.js]: else is not really necessary (just nitpicking)

File: frontend/app/components/crud-list/template.hbs [hunk 2]
```diff
@@ -12,29 +12,46 @@
...
+                <button class="btn btn-primary" {{action 'save' index entry}}><i class="fa fa-floppy-o"></i>&nbsp;{{t 'global.save'}}</button>
+                <button class="btn" {{action 'delete' index}}><i class="fa fa-times"></i>&nbsp;{{t 'global.delete'}}</button>
+              </td></tr>
+            {{/if}}
           {{else}}
-          <tr>
-            <td colspan="3" style="text-align: center">There's nothing to show here - yet!</td>
-          </tr>
+            <tr>
+              <td colspan="3" style="text-align: center">There's nothing to show here - yet!</td>
```
Comment (kaldras) [inline frontend/app/components/crud-list/template.hbs]: Add a translation for this, ty :)

File: frontend/app/vault/edit/template.hbs [hunk 3]
```diff
@@ -10,4 +10,4 @@
   model=model.details.secret
   onUpdate=(route-action 'save')}}
 
-<p><a class="color-danger" href="javascript:void(0)" {{action (route-action 'deleteNode')}}>{{t 'vault.delete-entry'}}</a></p>
+<!-- <p><a class="color-danger" href="javascript:void(0)" {{action (route-action 'deleteNode')}}>{{t 'vault.delete-entry'}}</a></p> -->
```
Comment (kaldras) [inline frontend/app/vault/edit/template.hbs]: Do you want to keep this one?

File: frontend/app/components/crud-list/component.js (hunk 1 above)
Comment (czosel) [inline frontend/app/components/crud-list/component.js]: true, fixed

File: frontend/app/components/crud-list/template.hbs (hunk 2 above)
Comment (czosel) [inline frontend/app/components/crud-list/template.hbs]: done

File: frontend/app/vault/edit/template.hbs (hunk 3 above)
Comment (czosel) [inline frontend/app/vault/edit/template.hbs]: Removed it

**Pull request merged**